Sources/Apodini/Modifier/DefaultRelationshipModifier.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Created by Andreas Bauer on 16.01.21.
+//
+
+struct DefaultRelationshipContextKey: OptionalContextKey {
+    // We really only need to check if the context key exists on a Endpoint
+    // So the type is actually irrelevant, but Bool probably fits best?
+    typealias Value = Bool
```

Comment:
I would propose to use `Void` as the `Value` as this indicates even more that there is no additional context carried with this `OptionalContextKey`:
```suggestion
    // This OptionalContextKey doesn't carry any additional context. 
    // We only need to check if the context key exists on a Endpoint
    typealias Value = Void
```
An other Apodini developer wound not be able to make an error and creating a `DefaultRelationshipContextKey` with the `value` being `false`, leading to an undefined behavior.
Checking if it is present is also a bit more descriptive:
```swift
let markedDefault = endpoint.context.get(valueFor: DefaultRelationshipContextKey.self) != nil
```

Reply:
Solved via 5ac7d7df3c2c1de6c3fd59031fa9f758d6822218